### ISSUE TITLE
Multiple QR Code

### ISSUE THREAD

**Issue by drakeman** [matyasf/zbar-ane#8]
Hello, thank you for this ANE, i already try and develop some sample app and it worked great.
Furthermore i would like to ask is it possible to detect multiple QR code in one scan ?? 

Thanks.

**Comment by matyasf** [comment]
Yes its possible but it would need a big refactoring. The native library supports it (see https://github.com/matyasf/zbar-ane/blob/master/java/src/net/sourceforge/zbar/ImageScanner.java ), but I shut down the native app as soon as I get a scan result. This would need that someone implements a continuous scan mode (alongside of the current "return from the app as soon as something is scanned" mode) which the user can close programatically. Sadly I dont have time for this in the near future.

**Comment by drakeman** [comment]
Waww thats a bit dissapointment for me, i guess i have to try to do it myself then or probably try another qr libraries. Thx for the quick reply, but i still like your lib so i'll would still using it for the simpler apps.

**Comment by matyasf** [comment]
If you decide to do it yourself feel free to ask for help or make a pull request!
